src/TestUtils/src/DeviceTests/AssertionExtensions.Android.cs

```diff
@@ -335,7 +335,17 @@ public static async Task<T> AttachAndRun<T>(this AView view, Func<Task<T>> actio
 				{
 					LayoutParameters = new FrameLayout.LayoutParams(ViewGroup.LayoutParams.MatchParent, ViewGroup.LayoutParams.MatchParent)
 				};
-				view.LayoutParameters = new FrameLayout.LayoutParams(view.Width, view.Height)
+
+				var width = view.Width;
+				var height = view.Height;
+
+				if (width <= 0)
+					width = FrameLayout.LayoutParams.WrapContent;
+
+				if (height <= 0)
+					height = FrameLayout.LayoutParams.WrapContent;
+
+				view.LayoutParameters = new FrameLayout.LayoutParams(width, height)
```

Comment:
I think setting this to view.width/view.height was causing some weird thrashing. Now when you run the tests you see them popup on the screen. The frame tests would layout the frame thing snap back to zero.